modules/videoio/src/cap_aravis.cpp

```diff
@@ -76,7 +82,7 @@
 #define MODE_Y800   CV_FOURCC_MACRO('Y','8','0','0')
 #define MODE_Y12    CV_FOURCC_MACRO('Y','1','2',' ')
 
-#define BETWEEN(a,b,c) ((a) < (b) ? (b) : ((a) > (c) ? (c) : (a) ))
+#define LIMIT(a,b,c) (cv::max(cv::min((a),(c)),(b)))
```

Comment:
[lets call this clip](https://docs.scipy.org/doc/numpy/reference/generated/numpy.clip.html)